### PR TITLE
v1 page updates 

### PR DESCRIPTION
**Pull request by tabern**
Updated index.html to support Blox v1 updates
Added Blox favicon

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,132 +46,55 @@
...
+				<div class="col-md-10">
+					<p class="block-description">Blox v1.0 is under active development.
 				</div>
 			</div>
-
 			<div class="custom-panel">
 				<div class="row">
 					<div class="col-sm-3">
-						<h3><img src="assets/img/repo.svg" /> <a href="https://github.com/aws/amazon-ecs-cli">ecs-cli</a></h3>
+						<h3><img src="assets/img/command-line-tools.svg" width="17px"/>&nbsp;&nbsp;&nbsp;<a href="https://github.com/blox/blox/blob/dev/README.md" target="_blank">Current Development</a></h3>
```
Comment (aaronkao) [inline index.html]: Shouldn't this be point here: https://github.com/blox/blox/projects/2 instead?

File: index.html [hunk 2]
```diff
@@ -46,132 +46,55 @@
...
-		<div class="container">
-
-			<h2><img src="assets/img/command-line-tools.svg" /> Command Line Tools</h2>
+			<h2><img src="assets/img/releases-and-roadmap.svg"/> Current Development</h2>
 
 			<div class="row">
-				<div class="col-md-7">
-					<p class="block-description">The following project provides local development tools for Amazon ECS.</p>
+				<div class="col-md-10">
+					<p class="block-description">Blox v1.0 is under active development.
```
Comment (aaronkao) [inline index.html]: What is current available for use? Nothing? What version number are we on right now?

File: index.html [hunk 3]
```diff
@@ -4,25 +4,25 @@
...
 					</p>
 				</div>
 			</div>
 
 			<div class="row">
-				<div class="col-sm-9">
-					<p class="block-description sublead">Blox gives you more control over how your containerized applications run on Amazon ECS.
-						It enables you to build schedulers and integrate third-party schedulers on top of ECS, while leveraging Amazon ECS to fully manage and scale your clusters.</p>
+				<div class="col-sm-9 tab">
+					<p class="block-description sublead">Blox provides scheduling capabilities that enable developers to have greater choice and flexibility over when to deploy, run, and scale tasks for containerized services.</p>
```
Comment (aaronkao) [inline index.html]: I am not sure this copy is correct. I don't understand what "greater choice" and "flexibility" means? What are we trying to get at? Its sort of a nebulous sentence.

**Comment by tabern** [comment]
Thanks @aaronkao - changes implemented in 0.9 commit.

File: index.html (hunk 3 above)
Comment (tabern) [inline index.html]: updated

File: index.html (hunk 2 above)
Comment (tabern) [inline index.html]: added copy to support this

File: index.html (hunk 1 above)
Comment (tabern) [inline index.html]: updated

**Pull request closed without merging**